omnipaxos/src/omni_paxos.rs

```diff
@@ -421,6 +421,11 @@ where
             cluster_state,
         }
     }
+
+    /// Returns whether Sequence Paxos instance is in Accept Phase.
+    pub fn is_accepted(&self) -> bool {
+        self.seq_paxos.get_state().1 == Phase::Accept
+    }
```

Comment:
I think it would be better to extend the [get_current_leader()](https://github.com/haraldng/omnipaxos/blob/master/omnipaxos/src/omni_paxos.rs#L272) function to return `Option<(NodeId, Phase)>`

Reply:
As I understand it, an Omnipaxos instance is only aware of its own `Phase`, not the leader’s. Returning `Phase` in the `get_current_leader` method may imply that `Phase` represents the leader, but that’s not accurate. Isn't?

Reply:
If the follower is in the `Accept` phase then that also applies to the leader. So one solution could be to change the return type to `Option<(NodeId, bool)>` and only return `true` if our own phase is `Accept`.

Reply:
Made changes as suggested, also added a doc comment, please feel free to suggest any changed to make that clearer.